File: TYPESCRIPT PROGRAMMING/Test70.ts
// 70. AUTOMORPHIC NUMBER
const Automorphic = (num: number): void => {
    if(num < 0) num = -num;
    let squareNum: number = num * num;
    let temp: number = num, count: number = 0;
    while(temp > 0) {
        count++;
        temp = Math.floor(temp / 10);
    }
    let lastdigit:  number = squareNum % Math.floor(Math.pow(10, count));
    if(lastdigit === num) console.log(`${num} Is An Automorphic Number`);
    else console.log(`${num} Is Not An Automorphic Number.`);
}
Automorphic(25);
Automorphic(37);

File: JAVA PROGRAMMING/Test70.java
// 70. AUTOMORPHIC NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Num Value: ");
        int a = in.nextInt();

        if (a < 0)
            a = -a;
        int squareNum = a * a;
        int temp = a;
        int count = 0;
        while (temp > 0) {
            count++;
            temp = temp / 10;
        }
        int lastDigit = squareNum % (int) Math.pow(10, count);

        if (lastDigit == a)
            System.out.print("Automorphic");
        else
            System.out.print("Not Automorphic");

        in.close();
    }
}